includes/Modules/Analytics.php

```diff
@@ -384,12 +384,13 @@ protected function print_amp_gtag() {
 			'vars' => array(
 				'gtag_id' => $tracking_id,
 				'config'  => array(
-					$tracking_id => array(
+					$tracking_id      => array(
 						'groups' => 'default',
 						'linker' => array(
 							'domains' => array( $this->get_home_domain() ),
 						),
 					),
+					'optoutElementId' => '__gaOptOutExtension',
```

Comment:
This has to be on the top-level, together with `vars`. See e.g. https://github.com/ampproject/amphtml/blob/b0aff1b9a59d3ca8f10a6ebe4d6eaeda8c826e61/extensions/amp-analytics/0.1/vendors/googleanalytics.json#L65